eng/mark-shipped.ps1

```diff
@@ -0,0 +1,51 @@
+[CmdletBinding(PositionalBinding=$false)]
+param ()
```

Comment:
There are no params, so there is no need for this IMHO. Is there a reason you added it?